source/versioned-api/versioned-api.rst

```diff
@@ -169,6 +169,16 @@ the specified value is equal to the server default. Drivers MUST NOT add any
 API versioning options if the user did not specify them.
 
 
+Handshake behavior
+~~~~~~~~~~~~~~~~~~
+
+Since the legacy hello command is not part of the versioned API, drivers MUST
```

Comment:
It's not clear that this requirement only applies when an API version is declared.  Can we prefix with

> If an API version was declared

as in the previous paragraph?

Reply:
I appreciate your motivation for calling isMaster "legacy hello", but I think "isMaster" is much clearer to future readers. Could you say "isMaster" here? Or if necessary, define "legacy hello" as "IsMaster" once before continuing to use "legacy hello".

Reply:
Also, do we plan to update handshake.rst to replace isMaster with hello there? I see that we updated a bunch of specifications this morning 99c9ee188a4bc4fc095837ae0be2451c30761c9b

Reply:
These will be changed as part of DRIVERS-1293.

Reply:
I've asked @JamesKovacs for clarification. IIRC, we intended for one spec to define the term "legacy hello" so we can link to it from other specs. I'll update this once I know what to link to.

Reply:
See https://github.com/mongodb/specifications/pull/975.